src/output.c

```diff
@@ -1002,8 +1006,16 @@ void OutputRegisterRootLogger(ThreadInitFunc ThreadInit,
     TAILQ_INSERT_TAIL(&RootLoggers, logger, entries);
 }
 
+//XXX FIXME  - What the best place and names for this
+void OutputLibEventUsePthreads() {
+#ifdef HAVE_LIBEVENT_PTHREADS
+    evthread_use_pthreads();
+#endif
+}
+
```

Comment:
Perhaps do this in util-log-redis.c, SCLogRedisInit(), then you can do the ifdef in there, and keep the libevent stuff out of src/output.c as libevent usage is only for redis, for now.